addons/hr_contract/wizard/hr_departure_wizard.py

```diff
@@ -8,6 +8,12 @@
 class HrDepartureWizard(models.TransientModel):
     _inherit = 'hr.departure.wizard'
 
+    def _get_employee_departure_date(self):
+        expired_contract = self.env['hr.contract'].search([('employee_id', '=', self.env.context['active_id']), ('state', '=', 'close')])
```

Comment:
There might be multiple contracts! If that's the case, this method would lead to a traceback. We should only take the latest one :)
```suggestion
        expired_contract = self.env['hr.contract'].search(
            [('employee_id', '=', self.env.context['active_id']), ('state', '=', 'close')],
            limit=1, order='date_end desc')
```

Reply:
> There might be multiple contracts! If that's the case, this method would lead to a traceback. We should only take the latest one :)

Right, I thought there would be only one contract in the Expired stage. Now I  checked, may be more than one, date will be different.Thank you.